lib/index.js

```diff
@@ -14,12 +14,11 @@ var httpAdapter = require('./adapters/http/http');
 PouchDB.adapter('http', httpAdapter);
 PouchDB.adapter('https', httpAdapter);
 
-PouchDB.adapter('idb', require('./adapters/idb/idb'));
-PouchDB.adapter('websql', require('./adapters/websql/websql'));
+PouchDB.adapter('idb', require('./adapters/idb/idb'), true);
+PouchDB.adapter('websql', require('./adapters/websql/websql'), true);
 PouchDB.plugin(require('pouchdb-mapreduce'));
 
 if (!process.browser) {
   var ldbAdapter = require('./adapters/leveldb/leveldb');
-  PouchDB.adapter('ldb', ldbAdapter);
```

Comment:
^ I don't remember seeing anywhere where this `'ldb'` name was used. I think we can get rid of it.